ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form-tab-container.view.js

```diff
@@ -18,6 +18,7 @@
  var template = require('./search-form.collection.hbs');
  var SearchFormCollectionView = require('./search-form.collection.view');
  var CustomElements = require('js/CustomElements');
+ var ResultForm = require('component/result-form/result-form');
```

Comment:
I couldn't find this being used.  Is it here for side effects?

Reply:
Removed unused import